README.md

```diff
@@ -1,2 +1,232 @@
-# elixir-style-guide
-Our Elixir style guide
+<img src="https://jungsoft.io/static/media/jungsoft_logo.c44eaf52.png" width="300px"/>
+
+# Elixir Style Guide
+
+Welcome to Jungsoft's Elixir Style Guide. Sit down, have a cup of coffee and read this calmly. ☕
+
+## Indroduction
+
+The goal of this guide is to help our team to understand and
+follow code style best practices, maintaining a pattern.
+
+As this guide is an extension of the [Christopher Adams Elixir Style Guide](https://github.com/christopheradams/elixir_style_guide), we **highly recommend** reading it before you continue.
+
+# Table of Contents
+
+* [Alias Organization](#alias-organization)
+* ['With do' Organization](#with-do-organization)
+* [Maps](#maps)
+* [Lists](#lists)
+
+## Aliases
+
+### Comma
+
+Aliases with multiple lines should contains a comma in the last module.
+
+Example:
+
+```elixir
+# ❌ Bad
+alias Jungsoft.{
+  Module1,
+  Module2
+}
+
+
+# ✅ Good
+alias Jungsoft.{
+  Module1,
+  Module2,
+}
+```
+
+### Ordering
+
+The aliases should be ordered alphabetically.
+
+Example:
+
+```elixir
+# ❌ Bad
+alias Jungsoft.{
+  CModule,
+  BModule,
+  AModule,
+}
+
+
+# ✅ Good
+alias Jungsoft.{
+  AModule,
+  BModule,
+  CModule,
+}
+```
+
+### SubModules
+
+The aliases should contains only one 'sub-module'.
+
+Example:
+
+```elixir
+# ❌ Bad
+alias Jungsoft.{
+  Module1.SubModule1,
+  Module1.SubModule2,
+  Module2.SubModule1.SubSubModule1,
+  Module2.SubModule1.SubSubModule2,
+}
+
+
+# ✅ Good
+alias Jungsoft.Module1.{
+  SubModule1,
+  SubModule2,
+}
+alias Jungsoft.Module2.SubModule1.{
+  SubSubModule1,
+  SubSubModule2,
+}
+```
+
+[Back to top ⬆️](#table-of-contents)
+
+## 'With do' Organization
+
+### Space
+
+When `with do` contains multiple lines, it's necessary to add two blank spaces before the first parameter. This is only valid when editor tab size is **2**. If you followed the Jungsoft guidelines this should be your case. 😊
+
+Example:
+
+```elixir
+# ❌ Bad
+with true <- Jungsoft.function_1(),
+      false <- Jungsoft.function_2()
+do
+  ## DO SOMETHING
+end
+
+# ❌ Bad
+with true <- Jungsoft.function_1(),
+     false <- Jungsoft.function_2()
+do
+  ## DO SOMETHING
+end
+
+# ❌ Bad
+with  true <- Jungsoft.function_1() do
+  ## DO SOMETHING
+end
+
+
+# ✅ Good
+with  true <- Jungsoft.function_1(),
+      false <- Jungsoft.function_2()
+do
+  ## DO SOMETHING
+end
+
+# ✅ Good
+with true <- Jungsoft.function_1() do
+  ## DO SOMETHING
+end
+```
+
+### Blank lines
```

Comment:
Maybe this rule should be more generic? It's not specific to `with`, but we should avoid unnecessary blank lines everywhere. For example:

```elixir
# ❌ Bad
defmodule Module do
  @moduledoc false

  def func do

    1 + 1
  end

end

# ✅ Good
defmodule Module do
  @moduledoc false

  def func do
    1 + 1
  end
end
```